test_scripts/UpdateVideoStreamingCapabilities/001_5_GetCapabilities_invalid_AddVSC.lua

```diff
@@ -0,0 +1,54 @@
+
+---------------------------------------------------------------------------------------------------
+-- [[ Required Shared libraries ]]
+local common = require('test_scripts/UpdateVideoStreamingCapabilities/common')
+
+--[[ Local Variables ]]
+local appSessionId = 1
+
+local checks = { }
+
+checks.invalid_type = common.getVideoStreamingCapability(1)
+checks.invalid_type.additionalVideoStreamingCapabilities[1].preferredResolution.resolutionWidth = true -- invalid type
+
+checks.invalid_value = common.getVideoStreamingCapability(2)
+checks.invalid_value.additionalVideoStreamingCapabilities[1].supportedFormats[1].codec = "H266" -- invalid value
+
+checks.invalid_nested_type = common.getVideoStreamingCapability(2)
+checks.invalid_nested_type.additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(1)
+checks.invalid_nested_type.additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[1].hapticSpatialDataSupported = 18 -- invalid type
+
+checks.invalid_nested_value = common.getVideoStreamingCapability(3)
+checks.invalid_nested_value.additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(2)
+checks.invalid_nested_value.additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[2].scale = -1 -- invalid value
+
+checks.invalid_deep_nested_type = common.getVideoStreamingCapability(1)
+checks.invalid_deep_nested_type.additionalVideoStreamingCapabilities[1] = common.getVideoStreamingCapability(3)
+checks.invalid_deep_nested_type.additionalVideoStreamingCapabilities[1].additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(2)
+checks.invalid_deep_nested_type.additionalVideoStreamingCapabilities[1].additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[2].supportedFormats = 2 -- invalid type
+
+checks.invalid_deep_nested_value = common.getVideoStreamingCapability(3)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(1)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[3] = common.getVideoStreamingCapability(5)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[3].additionalVideoStreamingCapabilities[2] = common.getVideoStreamingCapability(1)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[3].additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[1] = common.getVideoStreamingCapability(3)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[3].additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[1].additionalVideoStreamingCapabilities[1] = common.getVideoStreamingCapability(2)
+checks.invalid_deep_nested_value.additionalVideoStreamingCapabilities[3].additionalVideoStreamingCapabilities[2].additionalVideoStreamingCapabilities[1].additionalVideoStreamingCapabilities[1].additionalVideoStreamingCapabilities[2].pixelPerInch = -2 -- invalid value
```

Comment:
Do we really need so many checks for nested structs?
In my opinion, it's enough to check one level in a nested struct to make sure that existed logic works as expected.

Reply:
Done (see d0f84bf)